docs/UserGuide.md

```diff
@@ -1414,6 +1414,89 @@ Command Options: This command does not take in any arguments and will not proces
 
 #### 4.4.1 HelpCommand
 
+Get help for UniCa$h.
+
+Command: `help <COMMAND_WORD>`
+
+Command Argument: `<COMMAND_WORD>` is the command to get help for. If no 
+argument is specified, a general help message is shown as well as a pop up 
+containing a link to our User Guide.
```

Comment:
Think we can make it `[<COMMAND_WORD>]` to follow the convention of optional arguments